### ISSUE TITLE
Fallback to global or bundled remark version

### ISSUE THREAD

**Issue by bennypowers** [unifiedjs/unified-language-server#31]
### Initial checklist

- [X] I read the [support docs](https://github.com/unifiedjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/unifiedjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/unifiedjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aunifiedjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Problem

A user wishes to use `unified-language-server` or `remark-language-server` without installing remark in their local project folder

### Solution

`unified-language-server` should fall back to a global or bundled installation of remark if a local one is not found

### Alternatives

installing `remark` locally solely for the sake of a specific class of editors is not ideal

**Comment by wooorm** [comment]
This project allows that. Pass `defaultProcessor`.

**Comment by remcohaszing** [comment]
Yes, this project allows that, but `remark-language-server` doesn’t use it.

I’m curious to the scenario. Why does this user wish to use `remark-language-server` without installing `remark` in their local project folder?

I could imagine a number of scenarios, e.g.: Does the project use remark or `remark-cli`? Does the user want to use `remark-language-server`, but doesn’t the project use remark? Does the project use pnpm?

**Comment by bennypowers** [comment]
Here's one scenario, which I imagine is probably very common:
- The repo contains markdown files
- The repo does not use `remark` in any of it's app code or dependencies
- The user wants to format markdown files in the repo
- The user does not want to install any extra dependencies

**Comment by bennypowers** [comment]
At the moment, neovim users who follow the docs all the way down the rabbit hole, and end up installing the remark server, are going to get this error notification on every. single. character. they. type. in. a. markdown. file.
